static/app/views/replays/replays.tsx

```diff
@@ -247,12 +258,6 @@ const StyledPanelItem = styled('div')`
   margin-top: ${space(0.75)};
 `;
 
-const ReplayUserBadge = styled(UserBadge)`
-  font-size: ${p => p.theme.fontSizeLarge};
-  font-weight: 400;
-  line-height: 1.2;
-`;
-
```

Comment:
Turned out this wasn't doing anything anymore

All the props are overridden by the two spans that render (screen shot is for `<StyledEmail>`, but `<StyledName>` does the same):
<img width="347" alt="Screen Shot 2022-04-28 at 3 20 08 PM" src="https://user-images.githubusercontent.com/187460/165858577-f275513e-f65e-4656-bd2d-ed4fd51dc2aa.png">